universal-login-react/src/core/utils/getErc721Tokens.ts

```diff
@@ -0,0 +1,75 @@
+import {fetch} from '@unilogin/commons';
+
+interface IBasicToken {
+  tokenID: string;
+  contractAddress: string;
+}
+
+const OPENSEA_API_HTTP = 'https://api.opensea.io/api/v1/asset';
+
+const ETHERSCAN_API_HTTP = 'http://api.etherscan.io/api?module=account&action=tokennfttx&startblock=0&endblock=999999999&sort=asc&apikey=51GC96WCST6YU1BKYWGGX3PIT21UJPTFTE';
+
+export interface IErc721Token {
+  id: string;
+  name: string;
+  image: string;
+  description: string;
+  backgroundColor: string;
+  tokenName: string;
+  externalink: string;
+}
+
+const getApis = (network: string) => {
+  if (network === 'rinkeby') {
+    return [
+      ETHERSCAN_API_HTTP.replace('api', `api-${network}`),
+      OPENSEA_API_HTTP.replace('api', `${network}-api`),
+    ];
+  } else if (network !== 'mainnet') {
+    throw Error('Selected network is not valid');
```

Comment:
Extract error

Reply:
What do you think about the message `${network} is not supported by [api name]` or something like this?